daemons/fenced/fenced_remote.c

```diff
@@ -508,6 +532,25 @@ delegate_from_xml(xmlNode *xml)
     }
 }
 
+static void
+clear_check_async_reply(remote_fencing_op_t * op)
+{
+    check_async_reply_t *async_reply_op = NULL;
+
+    if (check_async_reply_list == NULL) {
+        return;
+    }
+
+    async_reply_op = g_hash_table_lookup(check_async_reply_list, op->id);
+    if (async_reply_op) {
+        if (async_reply_op->timer){
+            g_source_remove(async_reply_op->timer);
+            async_reply_op->timer = 0;
+        }
+        g_hash_table_remove(check_async_reply_list, async_reply_op->remote_op_id);
+    }
```

Comment:
Okay!

Reply:
Hi Hideo,

Actually, you don't even need the lookup. g_hash_table_remove() is fine if the entry doesn't exist (it just ignores it).

It doesn't matter if op->id is not the same pointer, since this was created with pcmk__strkey_table(), which compares the key as a string and not as a memory address.

Reply:
Okay!
Thanks!